tests/sfbase.py

```diff
@@ -828,6 +836,14 @@ def expected_metadata():
             'WorkStepTemplateShare': incremental_last_modified,
         }
 
+    def expected_streams(self):
+        """A set of expected stream names"""
+        streams = set(self.expected_metadata().keys())
+
```

Comment:
I implemented this the opposite way in my branch.  I should probably switch it back to match this since other other base file uses it too.  Let's discuss.